crates/sparrow-runtime/src/execute/operation/scan.rs

```diff
@@ -523,11 +522,11 @@ mod tests {
         let metadata_parquet_file = write_parquet_file(&metadata, None);
 
         let prepared = PreparedFile {
-            path: parquet_file.to_string_lossy().to_string(),
+            path: format!("file://{}", parquet_file.display()),
```

Comment:
Why does this need to be prefixed everywhere now? And is there a better way you can think of so we won't forget to leave it in the future? 

With a lot of extra work we could make a `LocalPath` wrapper that enforces the `file://` prefix. Or just change variable names to be prefixed with `local` to give us a hint to use `file://`. 

Not a blocker on this PR obviously

Reply:
Previously: We expected these to be local paths and just read them relative the current working directly. But it's worth noting that they were always local paths because we downloaded the files first, and were getting the path after the download. Now, we are getting the path at which we stored the file in S3, so we expect this to be a URL more often. 

In general, it seems like it is better to just expect a URL at this point in the code. The issues are what local paths be relative to? The root directory? The current working directory? etc. I could be convinced that we should handle this at this point, but it seems like it may be better for the client and/or API to handle that).